src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
```

Comment:
Node implemented arrow functions, but not contextual function name, so all arrow functions end up unnamed even though it should have received a contextual name.

Basically, if you enable `transform-es2015-function-name` in an environment, you also need to enable `transform-es2015-arrow-functions`, as the function name transform can only work on function expressions. The expected result is similar to `const x = function x() {}`.